Packages/com.unity.inputsystem/Extras/DualShock/Unity.InputSystem.DualShock.asmdef

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Unity.InputSystem.DualShock",
+    "references": [
+        "Unity.InputSystem"
```

Comment:
Was wondering... should the DLLs be named UnityEngine.XXX or Unity.XXX? The thing is that nothing in this package goes in the new Unity namespace and likely nothing ever will. At this point it seems there will be a new package which adds an ECS-compatible system in the Unity namespace. Seems more fitting to name that one Unity.Input or something and use UnityEngine.InputSystem for this one. Not sure what the convention is, though.

Reply:
According to the package template it is Unity.XXX (I just changed my package).  I think this is a new change going forward from the scripting team to move everything into Unity.XXX namespace eventually.
https://gitlab.internal.unity3d.com/upm-packages/upm-package-template/blob/master/Tests/Runtime/Unity.YourPackageName.Tests.asmdef